src/privateer/cpp/privateer-lib.h

```diff
@@ -129,8 +133,12 @@ namespace privateer
             return str.str();
         }
 
-        inline const bool sugar_contains_issues ( clipper::MSugar sugar )
+        inline const bool sugar_contains_issues ( clipper::MSugar sugar, clipper::MGlycan glycan  )
```

Comment:
Similarly here, params should be passed by reference i.e. 
`clipper::MSugar& sugar, clipper::MGlycan& glycan`